webapp/webpack.config_ict.js

```diff
@@ -14,6 +13,11 @@ module.exports = function (env) {
             path: path.resolve(__dirname, '../chromeextension'),
             filename: '[name]-bundle.js',
         },
+        performance: {
+            maxEntrypointSize: 900000, // 90KB
+            maxAssetSize: 900000 // 90KB
+        },
+        mode: env.production ? 'production' : 'development',
```

Comment:
What happens if the bundle goes over this size ? Will it just be a warning, I assume this is to suppress the warning.

Reply:
Yes. It 's just a warning. I did not configure it to error out if the budget was not met